tests/_data/snapshots/plain_alternative-package-registry.json.bin

```diff
@@ -103,7 +103,8 @@
           "type": "website",
           "comment": "as detected from PackageJson property \"homepage\""
         }
-      ]
+      ],
+      "evidence": {}
```

Comment:
Should only be added if `--gather-license-texts` was given.

Reply:
oops. fixed via 1d80836827f15935fb7b57858d8fba96bddec173